iceoryx_examples/icedelivery/iox_subscriber_untyped.cpp

```diff
@@ -21,12 +21,12 @@
 #include <csignal>
```

Comment:
Please remove ``#include "iceoryx_posh/popo/user_trigger.hpp"`` and ``#include "iceoryx_posh/popo/wait_set.hpp"``.

Reply:
Fixed